contracts/RiskManagerV1.sol

```diff
@@ -126,6 +135,15 @@ contract RiskManagerV1 is Auctioneer, Ownable {
         earlyCloseAuction(auction);
     }
 
+    /// @notice Updates collateralization threshold level.
+    /// @dev Can be called by the owner only.
+    /// @param _collateralizationThreshold New collateralization threshold level in percent.
+    function updateCollateralizationThreshold(
+        uint256 _collateralizationThreshold
+    ) external onlyOwner {
+        collateralizationThreshold = _collateralizationThreshold;
+    }
```

Comment:
This value needs to be updatable in two steps with a governance delay. Is this something we want to add in this PR or open another one?

Reply:
I started adding it in this PR. What should be the governance delay in this case, is it the same as for auction length? There might be a situation when BTC/ETH ratio changes very quickly and not sure if 12 hours is okay to change collateralization threshold. Maybe we should allow the governance to react faster than 12 hours? BTC/ETH ratio is one of the factors that tells us when it is still profitable to buy a deposit outside this coverage pool.

Reply:
404eebe

Reply:
I do not think the current ETH:BTC price ratio affects this threshold that much. 🤔 But let's discuss it separately in the thread about the correct initial value.